features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/tracking/ActivityLifecycleTrackingStrategy.kt

```diff
@@ -187,6 +187,20 @@ abstract class ActivityLifecycleTrackingStrategy :
         }
     }
 
+    private val Intent.safeExtras: Bundle?
+        get() = try {
+            // old Androids can throw different exceptions here making native calls
+            extras
+        } catch (@Suppress("TooGenericExceptionCaught") e: Exception) {
```

Comment:
> [!NOTE]
> Do we know what kind of exception could be thrown, to narrow it down?

Reply:
I saw `java.lang.NullPointerException` (from android.os.BaseBundle.<init>) and `java.lang.IllegalArgumentException` (from android.os.Parcel.nativeAppendFrom), so nothing in common.